### ISSUE TITLE
gitaly not starting after 9.4 update

### ISSUE THREAD

**Issue by MikeyG** [gitlabhq/gitlab-recipes#309]
Not sure exactly what is going wrong, after update to gitlab 9.4 starting gitaly returns:

Jul 26 06:48:27 pissant gitlab-gitaly[29603]: Gitaly, version v0.21.2-20170726.000657
Jul 26 06:48:27 pissant gitlab-gitaly[29603]: Usage: /home/git/gitaly/gitaly [OPTIONS] configfile
Jul 26 06:48:27 pissant gitlab-gitaly[29603]:   -version
Jul 26 06:48:27 pissant gitlab-gitaly[29603]:             Print version and exit
Jul 26 06:48:27 pissant systemd[1]: gitlab-gitaly.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Jul 26 06:48:27 pissant systemd[1]: gitlab-gitaly.service: Unit entered failed state.
Jul 26 06:48:27 pissant systemd[1]: gitlab-gitaly.service: Failed with result 'exit-code'.

However, is I start gitaly via init.d it runs.

**Comment by exislow** [comment]
Have the same issue.

**Comment by madddi** [comment]
There's a output redirection in the ExecStart statement. As systemd doesn't fork a shell, this cannot work. I removed it on my systems and let Gitaly log to journald.

```
ExecStart=/home/git/gitlab/bin/daemon_with_pidfile /home/git/gitlab/tmp/pids/gitaly.pid /home/git/gitaly/gitaly /home/git/gitaly/config.toml (code=exited, status=0/SUCCESS)
```

**Comment by axilleas** [comment]
Should be fixed with dc937dc86b101e2562887a7b11c91cf8a8050dea

**Comment by MikeyG** [comment]
This fixed it for me.
